### PR TITLE
Added CRUD routes to controllers and services.  Updated CarDealerCont…

### PR DESCRIPTION
**Pull request by stevenpazur**
…ext for Prices

Please try running this to make sure new CRUD routes work in Swagger/Postman.  Thanks! :D

### CODE REVIEW

File: TestConsole2/TestConsole2.csproj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+</Project>
```
Comment (LWooler) [inline TestConsole2/TestConsole2.csproj:8]: test console shouldn't be committed

File: CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/UpdateVehicleSubmission_Tests.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            databaseContext.SaveChanges();
+            submission.TimeStamp = new DateTime(1999, 1, 13, 3, 57, 32, 11);
+            vehicleSubmissionsService.UpdateVehicleSubmission(submission);
+            //Then
+            var result = databaseContext.VehicleSubmissions.ToList()[0].TimeStamp;
+            result.Should().Be(new DateTime(1999, 1, 13, 3, 57, 32, 11));
+        }
+
+        [Fact]
+        public void TestName()
```
Comment (LWooler) [inline CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/UpdateVehicleSubmission_Tests.cs:40]: test doesn't have name

File: CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/DeleteVehicleSubmission_Tests.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            databaseContext.SaveChanges();
+            submission.TimeStamp = new DateTime(1999, 1, 13, 3, 57, 32, 11);
+            vehicleSubmissionsService.DeleteVehicleSubmission(submission);
+            //Then
+            var result = databaseContext.VehicleSubmissions.ToList().Count;
+            result.Should().Be(0);
+        }
+
+        [Fact]
+        public void TestName()
```
Comment (LWooler) [inline CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/DeleteVehicleSubmission_Tests.cs:40]: test needs name

File: CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/DeleteVehicleSubmission_Tests.cs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+using Microsoft.EntityFrameworkCore;
+using CarDealerAPIService.App.Data;
+using CarDealerAPIService.App.models;
+
+namespace CarDealerWebAPI.Tests.VehicleSubmissionIntegrationTests
+{
+    public class DeleteVehicleSubmission_Tests
+    {
+        [Fact]
+        public void UpdateVehicleSubmission_ShouldUpdateVehicle()
```
Comment (LWooler) [inline CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/DeleteVehicleSubmission_Tests.cs:16]: wrong test name, this is testing deletion

File: CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/AddVehicleSubmission_Tests.cs [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            };
+            var submission3 = new VehicleSubmissions() {
+                User = MyUser,
+                TimeStamp = new DateTime(12, 12, 12),
+                Vehicle = new Vehicle() {Make="Toyota", Model="Highlander", Year=1994, VinNumber="abc"}
+            };
+            vehicleSubmissionsService.AddVehicleSubmission(submission1);
+            vehicleSubmissionsService.AddVehicleSubmission(submission2);
+            vehicleSubmissionsService.AddVehicleSubmission(submission3);
+            var result = vehicleSubmissionsService.GetAllVehicleSubmissionsByUser("abc123").Count;
```
Comment (RexBrown122) [inline CarDealerWebAPI.Tests/VehicleSubmissionIntegrationTests/AddVehicleSubmission_Tests.cs:65]: Does there need to be a save changes?

**Review by LWooler — CHANGES_REQUESTED** [review]

**Review by RexBrown122 — APPROVED** [review]

**Review by LWooler — APPROVED** [review]

**Pull request merged**